src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -503,23 +603,25 @@ private function createNewInvoice(Order $order, Cart $cart, $isFreeShipping)
     /**
      * @param int $orderInvoiceId
      * @param Cart $cart
+     * @param Product $product
```

Comment:
this param doesn't match the new argument

Reply:
Fixed